object_detection/ks-app/vendor/kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -25,7 +25,7 @@
     else
       $.params.httpProxyImage,
 
-    serviceType: "ClusterIP",
+    serviceType: "NodePort",
```

Comment:
Is there a reason you are making this override in the the vendor directory rather than modifying the generated component (in components) to override it.

Reply:
Thanks. Remove this change.